tasks/cephfs/fuse_mount.py

```diff
@@ -68,6 +68,13 @@ def mount(self):
         run_cmd.extend(run_cmd_tail)
 
         def list_connections():
+            self.client_remote.run(
```

Comment:
Would it be correct to simply return [] if the directory is not there?  We shouldn't have to manually modprobe fuse